make-uniform/transitland_feed/standard_route_name_db.R

```diff
@@ -19,3 +19,173 @@ standard_routes <- data.frame()
 
 sf_muni <- read.csv(sf_muni_path) %>%
   rename_all(tolower)
+
+# sf_cols <- colnames(sf_muni %>% select_at(vars(contains("transfer"))))
+# sf_cols <- c(sf_cols, colnames(sf_muni %>% select_at(vars(contains("route")))))
+sf_muni <- sf_muni %>%
+  select_at(vars(contains("route"))) %>%
+  select_at(vars(-contains("lat"))) %>%
+  select_at(vars(-contains("lon"))) %>%
+  select_at(vars(-contains("code")))
+
+unique_routes <- sf_muni %>% 
+  gather(variable, entry) %>%
+  # select(entry) %>%
+  unique() %>% 
+  filter(entry != "") %>%
+  mutate(base_name = entry) %>%
+  mutate(operator = "") %>%
+  mutate(operator = ifelse(str_detect(entry, fixed("missing", ignore_case = TRUE)), "Missing", operator)) %>%
+  mutate(operator = ifelse(entry == "-", "Missing", operator)) %>%
+  mutate(operator = ifelse(str_detect(entry, "^[0-9]"), "MUNI", operator)) %>% 
+  mutate(operator = ifelse(str_detect(entry, "^[A-Z]+-"), "MUNI", operator)) %>% 
+  
+  mutate(base_name = ifelse(str_detect(entry, "^MUNI "), 
```

Comment:
Can certainly wait until you have a version that is working, but there's obviously a lot of redundant code here that would be better in a method.